_posts/2017/2017-10-05-codetalks-2017-back-on-the-big-screen.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "code.talks 2017 - Back on the big screen"
+date: "2017-10-05 10:30:00"
+image: blog-header/codetalks-2017.png
+categories: events
+authors: ["Mats", "Mathias"]
+---
+
+<style>
+
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+Recently it was time again for popcorn, nachos and code.
+Code.talks is back on the big screen at Cinemaxx Dammtor after they tried Kampnagel last year.
+As a result code.talks had to shrink to 1.700 people from 1.900 last year, which is still 200 people more than two years ago.
+
+And yes - again it proved that a cinema is a pretty neat location for a software development conference.
+It seems that cosy seats, popcorn, and nachos are things that most of the attendees seemed comfortable with.
+
+Here are our highlights of this year's code.talks edition.
```

Comment:
Maybe it's better to move this sentence under the headline "Highlights". This way you do not have two headlines without something inbetween. At this point the sentence "comes out of nowhere".